pegasus/sites.v3/code.org/public/minecraft.haml

```diff
@@ -1,16 +1,22 @@
 ---
+<%
+  mc_landing_2017 = !!DCDO.get('hoc_launch', {})['mc_landing_2017']
+  share_image = "mc_social_#{mc_landing_2017 ? 2017 : 2016}.jpg"
```

Comment:
Just tested this in the [fb debugger](https://developers.facebook.com/tools/debug/sharing/?q=staging.code.org%2Fminecraft). It's having trouble finding the image because we called it as `.jpg` when the file exists as a `.png`. Can we request as `.png` instead? Or, if we need to do image optimization, convert the social media image?

Reply:
Doesn't pegasus return a .jpg if you request it as such, even if the source file is a .png?

Reply:
Oh weird. I thought I was getting a 404 with the `.jpg` earlier, but `https://code.org/images/mc/mc_social_2016.jpg` works now. I probably typed it wrong. But still see an error on the debugger, so it uses the first video image.

![image](https://user-images.githubusercontent.com/1920530/32748329-cc2f9dee-c870-11e7-8fbc-9deaa48f6999.png)

Reply:
Brendan brought up a good point that it might be because staging was rebuilding. On production, it fetched the 2016 image fine.